src/cmap/connection.ts

```diff
@@ -769,7 +777,6 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
     } finally {
       this.dataEvents = null;
       this.messageStream.pause();
```

Comment:
why don't we throwIfAborted here anymore?

Reply:
throwing inside a finally overrides the error thrown from the try/catch